components/envelopes.js

```diff
@@ -217,11 +217,19 @@ exports.init = function (accountId, baseUrl, accessToken) {
      * @public
      * @function
      * @param {string} envelopeId - ID of envelope to get list of recipients from.
+     * @param {boolean} [include_tabs] - When true the tab information associated with the recipient is included in the response.
      * @param {function} [callback] - Returns the list of recipients in the form of function(error, response).
      * @returns {Promise} - A thenable bluebird Promise; if callback is given it is called before the promise is resolved
      */
-    getRecipients: function (envelopeId, callback) {
-      return getRecipients(accessToken, baseUrl, envelopeId).asCallback(callback);
+    getRecipients: function (envelopeId, include_tabs, callback) {
+      // handle the case where people omit the tabs
+      if (arguments.length === 2 && Object.prototype.toString.call(include_tabs) === '[object Function]') {
+        callback = include_tabs;
+        include_tabs = false;
```

Comment:
Please add a test to account for the logic handling when `include_tabs` is omitted.